includes/form.inc

```diff
@@ -295,30 +295,3 @@ function radix_date($variables) {
 
   return '<div' . backdrop_attributes($attributes) . '>' . backdrop_render_children($element) . '</div>';
 }
-
-/**
- * Implements theme_fieldset().
- */
-function radix_fieldset($variables) {
```

Comment:
This theme function is required (unless theme_fieldset is not required by Backdrop anymore?).

This renders fieldsets as Bootstrap Panels. See http://getbootstrap.com/components/#panels. Thanks.